src/routes/docs/components/banner.md

```diff
@@ -55,93 +59,121 @@ Use this free example to show a text message for announcement with a CTA link, a
+      class="h-6 mr-2"
+      alt="Flowbite Logo" />
+    <span class="self-center text-lg font-semibold whitespace-nowrap dark:text-white">Flowbite</span>
   </a>
   <p class="flex items-center text-sm font-normal text-gray-500 dark:text-gray-400">
     Build websites even faster with components on top of Tailwind CSS
   </p>
 </Banner>
 ```
 
-## Newsletter sign-up banner 
+## Newsletter sign-up banner
 
 This example can be used to encourage your website visitors to sign up to your email newsletter by showing an inline form inside the sticky banner on the top side of your page.
 
-```svelte example class="flex flex-col relative" hideScript
+```svelte example class="flex flex-col relative"
 <script>
-  import { Banner, Skeleton, ImagePlaceholder } from "flowbite-svelte"
+  import { Banner, Skeleton, ImagePlaceholder } from 'flowbite-svelte';
 </script>
 
-<Skeleton class="py-4"/>
-<ImagePlaceholder class="py-4"/>
+<Skeleton class="py-4" />
+<ImagePlaceholder class="py-4" />
 
 <Banner id="signup-banner" position="absolute" bannerType="signup">
   <form action="/" class="flex flex-col items-center w-full md:flex-row">
-    <label for="email" class="flex-shrink-0 mb-2 mr-auto text-sm font-medium text-gray-500 md:mb-0 md:mr-4 dark:text-gray-400 md:m-0">Sign up for our newsletter</label>
-    <input type="email" id="email" placeholder="Enter your email" class="bg-white border border-gray-300 text-gray-900 md:w-64 mb-2 md:mb-0 md:mr-4 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-600 dark:border-gray-500 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500" required />
-    <button type="submit" class="text-white bg-primary-700 hover:bg-primary-800 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800">Subscribe</button>
-</form>
+    <label
+      for="email"
+      class="flex-shrink-0 mb-2 mr-auto text-sm font-medium text-gray-500 md:mb-0 md:mr-4 dark:text-gray-400 md:m-0"
+      >Sign up for our newsletter</label>
+    <input
+      type="email"
+      id="email"
+      placeholder="Enter your email"
+      class="bg-white border border-gray-300 text-gray-900 md:w-64 mb-2 md:mb-0 md:mr-4 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-600 dark:border-gray-500 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
+      required />
+    <button
+      type="submit"
+      class="text-white bg-primary-700 hover:bg-primary-800 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
+      >Subscribe</button>
+  </form>
```

Comment:
This formatting (wrapping long lines) is discourage in the examples. It decrease the readability. See:
```svelte
 <label
      for="email"
      class="flex-shrink-0 mb-2 mr-auto text-sm font-medium text-gray-500 md:mb-0 md:mr-4 dark:text-gray-400 md:m-0"
      >Sign up for our newsletter</label>
    <input
      type="email"
      id="email"
      placeholder="Enter your email"
      class="bg-white border border-gray-300 text-gray-900 md:w-64 mb-2 md:mb-0 md:mr-4 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-600 dark:border-gray-500 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
      required />
    <button
      type="submit"
      class="text-white bg-primary-700 hover:bg-primary-800 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
      >Subscribe</button>
```
vs
```svelte
   <label for="email"  class="flex-shrink-0 mb-2 mr-auto text-sm font-medium text-gray-500 md:mb-0 md:mr-4 dark:text-gray-400 md:m-0">
     Sign up for our newsletter
    </label>
    <input  type="email"  id="email" placeholder="Enter your email" class="bg-white border border-gray-300 text-gray-900 md:w-64 mb-2 md:mb-0 md:mr-4 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-600 dark:border-gray-500 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"  required />
    <button type="submit" class="text-white bg-primary-700 hover:bg-primary-800 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800">
       Subscribe
   </button>
```

Reply:
Unfortunately it's in lots of places.

Reply:
We need to update .prettierrc config:

```
{
  "useTabs": false,
  "singleQuote": true,
  "trailingComma": "none",
  "printWidth": 109,
  "embeddedLanguageFormatting": "auto",
  "bracketSameLine": true
}
```

Reply:
I think increasing the `printWidth` value may work.

Reply:
Yes, but that must be set for markdown files only. In the normal code wrapping at 109 chars is a good thing.